src/payment/instrument/instrument-response-transformer.ts

```diff
@@ -41,11 +42,23 @@ export default class InstrumentResponseTransformer {
 
     private _transformVaultedInstruments(vaultedInstruments: InternalInstrument[] = []): PaymentInstrument[] {
         return vaultedInstruments
-            .map(instrument => this._isAccountInstrument(instrument) ? mapToAccountInstrument(instrument) : mapToCardInstrument(instrument) );
+            .map(instrument => {
+                if (this._isPayPalInstrument(instrument)) {
+                    return mapToPayPalInstrument(instrument);
+                } else if (this._isBankInstrument(instrument)) {
+                    return mapToBankInstrument(instrument);
+                }
```

Comment:
No need to chain the `if` statements:

```suggestion
                if (this._isPayPalInstrument(instrument)) {
                    return mapToPayPalInstrument(instrument);
                }
                
                if (this._isBankInstrument(instrument)) {
                    return mapToBankInstrument(instrument);
                }
```

Reply:
Done 👍